static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -139,6 +145,21 @@ function NativeFrame({
     frame.function !== frame.rawFunction;
 
   const [expanded, setExpanded] = useState(expandable ? isExpanded ?? false : false);
+  const [isHovering, setHovering] = useState(false);
+
+  const contextLine = (frame?.context || []).find(l => l[0] === frame.lineNo);
+  const hasStacktraceLink =
+    frame.inApp && !!frame.filename && frame.lineNo && (isHovering || expanded);
```

Comment:
One thing to follow up on here is that it looks like some frames have a filename but no lineno. We should remove the lineno requirement at some point, but I think it will require endpoint changes to support not having a lineno: 

![CleanShot 2023-12-21 at 11 55 17](https://github.com/getsentry/sentry/assets/10888943/e5e9b095-c6ae-4a2e-99ac-ea015abaf8eb)

See this issue: https://demo.sentry.io/issues/4059476887/